cpp/src/slice2java/Gen.h

```diff
@@ -117,7 +116,7 @@ namespace Slice
     class Gen final
     {
     public:
-        Gen(const std::string&, std::string, const std::vector<std::string>&, std::string);
```

Comment:
For some reason we were passing in the executable name (`slice2java`) here.
We never checked or used this field though. Dead.